### ISSUE TITLE
Bruk Aksel på "Kopier Lenke"

### ISSUE THREAD

**Issue by PerOlavM** [navikt/nav-enonicxp-frontend#1651]
CopyButton med aria-label

https://nav-it.slack.com/archives/C7MANSGLS/p1702286839450459

**Comment by PerOlavM** [comment]
Dette er en funksjonalitet er laget for  superbrukere i NAV. Løsning med Knapp krever JavaScript.

- Teamet ønsker å ta en diskusjon på nyåret om den skal "back to basics" ved å være en skjult funksjonlitet knyttet til selve headingen. 
- På kort sikt rulles dette tilbake til løsningen med en egen ankerlenke som ligger under heading, med TILLEGG av aria-label for å gjøre det lettere for brukere av skjermleser.
